인코딩 포맷 변경

Kkumteulyi opened the pull request:
## 개요
<!-- 어떤 점이 변경되었는지, 간단하게 작성해주세요. -->
- 코딩 포맷을 EUC-KR에서 UTF - 8로 변경하였습니다.
## 변경 사항
<!-- 어떤 점에 변경되었는지 상세하게 작성해주세요. -->
- 기존 스크립트 포맷: EUC - KR  => UTF - 8
## 참고 자료
<!-- 기능 개발에 참고한 자료가 있다면 작성해주세요. (필수는 아닙니다.) -->

Kkumteulyi committed:
선택지 수정

선택지 수정 사항
- 아이템이 존재하지 않을 경우 선택지 아래에 붉은 글씨로 표시
- 선택 불가는 구현 중
- 선택지 옆에 현재 선택된 선택지가 무엇인지 알려주는 표시 추가

Kkumteulyi committed:
C# 코드 포맷 변경

EUC-KR -> UTF - 8 로 변경

Kkumteulyi committed:
기타 수정 사항

- 코드에 주석으로 적은 한글 깨짐 수정
- 가방 위에 표시되던 글자 제거

miniron-v commented:
선택지 수정 PR을 올린 후에 추가 작업 사항이 있어 올리신 거 같은데... PR이 열려 있을 때 해당 브랜치에 커밋하면 PR이 갱신된다는 사실, 알고 계셨나요?

WooMH committed:
Merge branch 'develop' into feature/StoryInterface

WooMH committed:
Inventory ON/OFF 수정

가방 버튼에 포인터를 올리면 열리고 포인터를 내리면 닫히도록 수정하였습니다. 인벤토리 UI가 버튼을 가릴 때 버튼에 올리면 UI가 깜박거리기에 UI 위치도 변경하였습니다.

Kkumteulyi committed:
24.05.12 Story Interface 패치 노트

파일 형식 변경
- 기존 TXT 파일 대신 CSV 파일로 대체

플레이어 HP 틀 변경
- 플레이어의 HP 표시를 Slider를 사용하여 플레이어 HP 변수 값에 따라 바뀌도록 틀 구성
*HP 변수 값만 불러오면 바로 적용 가능*

특정 이벤트에서 아이템 추가
- Items.cs에서 아이템 추가 함수 구현 시 바로 적용 가능하게 틀 구성

기타 수정 사항
- PreAlpha에서 Merge하는 과정에서 들고 온 배틀 씬 삭제
